jstz_node/src/services/accounts.rs

```diff
@@ -0,0 +1,46 @@
+use actix_web::{
+    get,
+    web::{Data, Path, ServiceConfig},
+    HttpResponse, Responder, Scope,
+};
+use anyhow::anyhow;
+use jstz_crypto::public_key_hash::PublicKeyHash;
+use jstz_proto::context::account::Account;
+
+use crate::{rollup::RollupClient, Result};
+
+#[get("/{address}/nonce")]
+async fn nonce(
+    rollup_client: Data<RollupClient>,
+    path: Path<String>,
+) -> Result<impl Responder> {
+    let address = path.into_inner();
+
+    let address_path = Account::path(
+        &PublicKeyHash::from_base58(&address).expect("Failed to create address"),
+    )
+    .expect("Failed to get account path");
+
+    let value = rollup_client.get_value(&address_path.to_string()).await?;
```

Comment:
Perhaps for consistency? (+ quicker)
```suggestion
    let key = format!("/jstz_account/{}", path.into_inner());

    let value = rollup_client.get_value(&key).await?;
```